src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// write code here
+const headers = document.querySelectorAll('th');
+const body = document.querySelector('tbody');
+
+function sortTab(index) {
+  const rows = Array.from(body.querySelectorAll('tr'));
+
+  rows.sort((rowA, rowB) => {
+    const cellA = rowA.cells[index].textContent.trim();
+    const cellB = rowB.cells[index].textContent.trim();
+
+    return cellA.localeCompare(cellB, undefined, { numeric: true });
+  });
+
+  body.innerHTML = '';
```

Comment:
Clearing the `tbody` innerHTML and then appending sorted rows is correct, but be cautious as this approach will remove any event listeners attached to the rows. If rows have event listeners, consider using `DocumentFragment` to minimize reflows and retain event listeners.